lit/docs/install/worker.lit

```diff
@@ -239,41 +246,225 @@ decide much on its own.
 \section{
   \title{Configuring the \code{worker} node}{worker-configuration}
 
-  If there's something special about your worker and you'd like to target
-  builds at it specifically, you can configure tags like so:
+  \section{
+    \title{Tagging Workers}
+
+    If there's something special about your worker and you'd like to target
+    builds at it specifically, you can configure tags like so:
+
+    \codeblock{bash}{{
+    CONCOURSE_TAG="tag-1,tag-2"
+    }}
+
+    A tagged worker is taken out of the default placement logic. Tagged workers
+    will not be used for any untagged \reference{steps}.
+
+    To run build steps on a tagged worker, specify the
+    \reference{schema.step.tags} on any particular step in your
+    \reference{jobs}{job}.
+
+    To perform resource \code{check}s on on a tagged worker, specify
+    \reference{schema.resource.tags}{\code{tags}} on the resource declaration.
+  }
+
+  \section{
+    \title{Team Workers}
+
+    If you want to isolate \bold{all workloads} for a
+    \reference{managing-teams}{team} then you can configure a worker to belong
+    to a single team like so:
+
+    \codeblock{bash}{{
+    CONCOURSE_TEAM="lightweavers"
+    }}
+
+    Once an untagged team worker is registered Concourse will schedule all
+    untagged builds for that team on its team worker(s). Builds for this team
+    will not be scheduled on any untagged, non-team workers anymore.
+
+    It is possible to have a Concourse cluster made up of only team workers and
+    have zero non-team workers, though this is not a common setup because
+    resource uitilization across all workers ends up underuitilized. It is
+    useful though if you have a particular team with heavy worloads that
+    usually bothers other teams pipelines.
```

Comment:
```suggestion
    resource utilization across all workers ends up under-utilized. It is
    useful though if you have a particular team with heavy workloads that
    usually bothers other teams pipelines.
```

Reply:
Can you tell I forgot to `:set spell` here 😅